elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -97,12 +97,20 @@ public function filter__ep_index_name( $index_name, $blog_id, $indexables ) {
 
 	public function filter__ep_do_intercept_request( $request, $query, $args, $failures ) {
 		$fallback_error = new \WP_Error( 'vip-elasticsearch-upstream-request-failed', 'There was an error connecting to the upstream Elasticsearch server' );
+		$timeout = 2;
 
 		// TEMP - currently ES server is using a self signed cert during the testing phase...that'll be changed in the near
 		// future, at which time we can remove this
 		$args['sslverify'] = false;
-	
-		$request = vip_safe_wp_remote_request( $query['url'], $fallback_error, 3, 1, 20, $args );
+
+		// If query url ends with '_bulk'
+		$query_path = wp_parse_url( $query[ 'url' ], PHP_URL_PATH );
+		if ( 0 === substr_compare( $query_path, '_bulk', -5 ) ) {
```

Comment:
We could also use `wp_endswith( $query_path, '_bulk' )` for convenience
```suggestion
		if ( wp_endswith( $query_path, '_bulk' ) ) {
```